lib/dbservice/session_schedules.ex

```diff
@@ -0,0 +1,81 @@
+defmodule Dbservice.SessionSchedules do
+  @moduledoc """
+  The SessionSchedules context.
+  """
+
+  import Ecto.Query, warn: false
+  alias Dbservice.Repo
+
+  alias Dbservice.Sessions.SessionSchedule
+
+  @doc """
+  Returns the list of session_schedule.
+  ## Examples
+      iex> list_session_schedule()
+      [%SessionSchedule{}, ...]
+  """
+  def list_session_schedule do
+    Repo.all(SessionSchedule)
+  end
+
+  @doc """
+  Gets a single session_schedule.
+  Raises `Ecto.NoResultsError` if the session_schedule does not exist.
+  ## Examples
+      iex> get_session_schedule!(123)
+      %SessionSchedule{}
+      iex> get_session_schedule!(456)
+      ** (Ecto.NoResultsError)
+  """
+  def get_session_schedule!(id), do: Repo.get!(SessionSchedule, id)
```

Comment:
`id` here refers to the internal db id, right?

Can we add a method to get session schedule based on batch id?

Reply:
It's already implemented. We can just use `batch_id` as a query parameter something like:
`/api/session-schedule?batch_id=1`